_includes/neurips/2020/about.neurips.html

```diff
@@ -349,10 +349,10 @@ <h2 style="text-align: center">Organizers</h2>
 
       <h3>Workshop Chairs</h3>
       <ul>
-        <!-- <li><em>General & Logistics Co-Chairs --</em> Maria Luisa Santiago (Accel AI)</li> -->
+        <li><em>General & Logistics Co-Chairs --</em> Maria Luisa Santiago (Accel AI)</li>
```

Comment:
This position was removed during the last push, please remove again. Please make sure to pull the latest master branch from the main repository prior to making changes to your fork.